### PR TITLE
audio: cleanup in components

### PR DESCRIPTION
**Pull request by tlauda**
Set of patches to cleanup audio components:
- Put multifile components in directories.
- Fix checkpatch errors.
- Remove check for MODULE_TEST definition.

### CODE REVIEW

File: src/audio/CMakeLists.txt [hunk 1]
```diff
@@ -121,8 +101,8 @@ check_optimization(hifi3 -mhifi3 -DOPS_HIFI3)
 set(sof_audio_modules volume src)
 
 # sources for each module
-set(volume_sources volume.c volume_generic.c)
-set(src_sources src.c src_generic.c)
+set(volume_sources volume/volume.c volume/volume_generic.c)
```
Comment (jajanusz) [inline src/audio/CMakeLists.txt]: Would be better to not handle f.e. volume.c on 2 different levels for different builds. We should have some kind of function here that is called in subdirs maybe. I can refactor that part later. We can merge it like it is now cos it's step in the good direction and it's better to have smaller PRs anyway.

File: src/audio/eq_fir/eq_fir.c [hunk 2]
```diff
@@ -613,7 +614,9 @@ static int fir_cmd_set_data(struct comp_dev *dev,
 		/* Just copy the configuration. The EQ will be initialized in
 		 * prepare().
 		 */
-		memcpy(dst + offset, src, cdata->num_elems);
+		assert(!memcpy_s(dst + offset, cdata->num_elems +
+				 cdata->elems_remaining - offset, src,
+				 cdata->num_elems));
```
Comment (plbossart) [inline src/audio/eq_fir/eq_fir.c]: maybe put the change to memcpy_s as a separate patch, this has nothing to do with check patch?

Comment (tlauda) [reply src/audio/eq_fir/eq_fir.c]: @plbossart memcpy_s requirement has been recently added to checkpatch, but sure, I think the separate patch would be also OK.

Comment (tlauda) [reply src/audio/eq_fir/eq_fir.c]: Done.

**Comment by jajanusz** [comment]
@tlauda Please fix ut
